analyzers/golang/analyze.go

```diff
@@ -90,11 +162,6 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 			return graph.Deps{}, err
 		}
 
-	// Resolve revisions by traversing the local $GOPATH and calling the package's
```

Comment:
Removed because the code is listed as not implemented and we don't plan on implementing this.